Source/WebCore/rendering/RenderElement.h

```diff
@@ -68,6 +69,7 @@ class RenderElement : public RenderObject {
     bool hasInitializedStyle() const { return m_hasInitializedStyle; }
 
     const RenderStyle& style() const { return m_style; }
+    const CheckedRef<const RenderStyle> checkedStyle() const { return m_style; }
```

Comment:
@rniwa It doesn't seem like this should be needed? m_style is a plain data member, it stays alive as long as the RenderElement.